buildSrc/src/main/kotlin/com/datadog/gradle/config/KotlinConfig.kt

```diff
@@ -17,4 +19,20 @@ fun Project.kotlinConfig() {
             jvmTarget = JavaVersion.VERSION_1_8.toString()
         }
     }
+
+    val moduleName = this@kotlinConfig.name
+    val javaAgentJar = File(File(rootDir, "libs"), "dd-java-agent-0.67.0.jar")
+    afterEvaluate {
+        taskConfig<AndroidUnitTest> {
+            if (environment["DD_INTEGRATION_JUNIT_5_ENABLED"] == "true") {
+                val variant = variantName.substringBeforeLast("UnitTest")
+
+                environment["DD_ENV_TESTS"]?.let { environment("DD_ENV", it) }
```

Comment:
can you please add a comment here about what these environment variables are doing ? The first 3 entries are a bit unclear to me.